.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,31 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-versions: [ '7.2', '7.3', '7.4', '8.0', '8.1' ]
```

Comment:
If 8.1 it is optionally requirements and looks like it not pass the tests should we remove it?